packages/usa-input-prefix-suffix/src/styles/_usa-input-prefix-suffix.scss

```diff
@@ -21,28 +24,52 @@
     @include u-border($theme-input-state-border-width, "success");
   }
 
+  &--error,
+  &--success {
+    & input:focus {
+      outline-offset: units($theme-input-state-border-width);
+    }
+  }
```

Comment:
Currently, the focus outline is placed outside of the succes/error border. This allows for the same styling

![image](https://user-images.githubusercontent.com/25211650/218498843-936632e3-a1d5-4f26-80d2-04f9203da40f.png)

If we prefer to have the focus cover the success/error border, we can remove these lines

Reply:
I think the the outline-offset is a good consideration. Nice call.

Reply:
I did notice that the input border (1px) is now covered by the focus outline, whereas it was visible before. It is a small thing, but just wanted to flag the visual difference. Not sure if it needs any action.